test/env/run_e2e_tests.sh

```diff
@@ -6,6 +6,19 @@ BUILD_SCM_BRANCH=${BUILD_SCM_BRANCH:-"master"}
 BUILD_SCM_REVISION=${BUILD_SCM_REVISION:-"latest"}
 SKIP_TEST=${SKIP_TEST:-no}
 
+if [[ $BUILD_SCM_REVISION != "latest" ]] ; then
+    while : ; do
+        check_suites=$(curl -s -H "Accept: application/vnd.github.antiope-preview+json" https://api.github.com/repos/Juniper/contrail-operator/commits/$BUILD_SCM_REVISION/check-suites)
+        status=$(echo "$check_suites" | jq -cr '.check_suites[]|select(.app.slug == "google-cloud-build").status')
+        [[ $status != "completed" ]] && echo "Waiting for upstream job. Current status: $status" && sleep 120 && continue
+        conclusion=$(echo "$check_suites" | jq -cr '.check_suites[]|select(.app.slug == "google-cloud-build").conclusion')
+        [[ $conclusion == "success" ]] && break
+        echo "Upstream job failed with conclusion: $conclusion"
```

Comment:
One more thing, at the first sight I agreed that e2e tests don't have to be run when the upstream job failed but are we sure that we don't want to know the status of e2e when for example only unit tests failed in GCB job? I know that it is possible to have a problem with phishing images but I'm not sure if we should stop e2e from running in case of failure od GCB job. I would like to know your opinion about that.